source/client-side-encryption/tests/README.rst

```diff
@@ -1007,4 +1007,57 @@ Drivers that do not support an unlimited maximum pool size MUST skip this test.
       - a find on "db"
    - Expect ``client_keyvault`` to have captured one ``CommandStartedEvent``:
       - a find on "keyvault".
-- ExpectedNumberOfClients: 1
\ No newline at end of file
+- ExpectedNumberOfClients: 1
+
+KMS TLS Tests
+~~~~~~~~~~~~~
+
+.. _ca.pem: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/x509gen/ca.pem
+
+The following tests that connections to KMS servers with TLS verify peer certificates.
+
+The two tests below make use of mock KMS servers which can be run on Evergreen using `the mock KMS server script <https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/mock_kms.js>`_.
```

Comment:
The links to the mock kms server script and wrong-host PEM won't work until the associated drivers evergreen PR is merged.